makefile

```diff
@@ -620,3 +620,8 @@ build-dev-jobservice:
 	docker build --build-arg APP_UID=$(shell id -u) --build-arg APP_GID=$(shell id -g) \
 		$(dockerFlags) -t armada-jobservice -f ./docs/dev/build/jobservice/Dockerfile ./.build/jobservice
 
+tests-dev:
```

Comment:
I think you should open this as separate PR.  This looks useful but unrelated to this PR?

Reply:
thanks will do